core/extensions.js

```diff
@@ -76,7 +73,7 @@ Blockly.Extensions.register = function(name, initFn) {
  *     registered.
  */
 Blockly.Extensions.registerMixin = function(name, mixinObj) {
-  if (!goog.isObject(mixinObj)){
+  if (!mixinObj || typeof mixinObj != 'object'){
```

Comment:
Can't this be replaced with just the RHS?

Reply:
Nope:  typeof null == 'object'